resources/js/components/SearchForm.vue

```diff
@@ -0,0 +1,161 @@
+<script>
+import Vue from 'vue';
+
+const moment = require('moment');
+require('moment/locale/de');
+import checkView from 'vue-check-view';
+Vue.use(checkView);
+
+export default {
+    name: 'SearchForm',
+    props: {
+        committees: {
+            type: Array,
+            default: () => []
+        }
+    },
+    data() {
+        return {
+            inputValue: '',
+            committeesList: this.committees,
+            sortedCommittees: [],
+            filtered: false
+        };
+    },
+    created () {
+        this.sortByChar();
+    },
+    methods: {
+        sortByChar() {
+            let char = '';
+            this.committeesList
+                .sort((a, b) => a.title === b.title ? 0 : +(a.title > b.title) || -1)
+                .forEach(el => {
+                    if (!this.sortedCommittees.length || char !== el.title[0].toLowerCase()) {
+                        char = el.title[0].toLowerCase();
+                        let arr = this.committeesList.filter(el => el.title[0].toLowerCase() === char);
+                        this.sortedCommittees.push({
+                            data: arr,
+                            char
+                        });
+                    }
+                });
+        },
+        filterList (e) {
+            let value = e.target.value;
+            if (value) {
+                this.sortedCommittees = this.committeesList
+                    .filter(el => el.title.toLowerCase().includes(value.toLowerCase()));
+                this.filtered = true;
```

Comment:
also i think if you use `inputValue` then i think you can remove `this.filtered` because then instead of checking `this.filtered` you can check if `inputValue.length > 0 && filteredResults.length > 0` which could be computed, this way everything will be more dependent on the data changes, so you don't have to actually mutate `this.filtered` by hand to change something on the page